packages/core/src/clients/ILeaderboardClient.ts

```diff
@@ -0,0 +1,20 @@
+import {InternalLeaderboardClient} from '../internal/clients';
+import {
+  LeaderboardFetchByRankCallOptions,
+  LeaderboardFetchByScoreCallOptions,
+  LeaderboardGetRankCallOptions,
+  LeaderboardOrder,
+} from '../utils';
+
+export interface ILeaderboardClient {
+  createLeaderboard(
```

Comment:
I think let's just call this `leaderboard`; it may already exist, and the word `create` implies some things.